NAMESPACE

```diff
@@ -1,16 +1,16 @@
 # Generated by roxygen2: do not edit by hand
 
-export(add_obs_error_to_nowcast)
-export(add_obs_errors_to_nowcast)
+export(add_obs_err_to_pt_nowcast_mat)
```

Comment:
Some naming thoughts... (not required maybe for this PR

point_to_nowcast_matrix
point_to_nowcast_matrix_list 

?

Reply:
with the theme of shortening could also making input implicit and call them 

add_uncertainty/dispersion or similar and add_<>_list ?